src/Tasks.js

```diff
@@ -81,24 +91,61 @@ const TaskCard = React.memo(function({task,editClick}) {
         </div>
     )
 });
+TaskCard.propTypes = {
+    task: PropTypes.shape({
+        taskId: PropTypes.number.isRequired,
+        imageUrl: PropTypes.string,
+        title: PropTypes.string,
+        assignee: PropTypes.number.isRequired,
+        status: PropTypes.oneOf[0,1,2,3],
+        stages: PropTypes.objectOf(PropTypes.bool),
+        dueDate: PropTypes.string,
+    }),
+    editClick: PropTypes.func,
+}
+
+function UserTasks({userTasks,editClick}) {
+    return(
+        <>
+            {userTasks.map(task => <TaskCard task={task} key={task.taskId} editClick={editClick}/>)}
+        </>
+    )
+}
 
 function UserColumn({userTasks,userName,editClickHandle,userId}) {
-    console.log(userTasks);
 
+    const [sort,setSort] = useState(1);
```

Comment:
why are sort values 1 and 2?

if only 1 and 2 are needed, then can use boolean
if not use proper strings to denote what they mean - ASC and DESC